pkg/api/routes.json

```diff
@@ -5,36 +5,36 @@
         "pkg": "github.com/go-chi/chi/middleware",
         "func": "RequestID",
         "comment": "RequestID is a middleware that injects a request ID into the context of each\nrequest. A request ID is a string of the form \"host.example.com/random-0001\",\nwhere \"random\" is a base62 random string that uniquely identifies this go\nprocess, and where the last number is an atomically incremented request\ncounter.\n",
-        "file": "github.com/go-chi/chi/middleware/request_id.go",
+        "file": "/home/lel/programming/pkg/mod/github.com/go-chi/chi@v4.0.2+incompatible/middleware/request_id.go",
```

Comment:
Is that expected to be a local path vs. a go package?

Reply:
To @florianl as well. No, this was not intentional. To be honest, I think we should drop the auto-generated API doc altogether and either write a proper README or document it in something like (swagger)[https://swagger.io/tools/open-source/].